### ISSUE TITLE
document._currentScript not needed

### ISSUE THREAD

**Issue by warpech** [Juicy/juicy-html#26]
This custom element has a variant of the following line:

```js
var script = document._currentScript || document.currentScript;
var template = script.previousElementSibling;
```

`document._currentScript` was only needed in IE11, but Starcounter does not support that browser anymore. 

Therefore, this code can be simplified to:

```js
var template = document.currentScript.previousElementSibling;
```

If we want to drop support for IE11.

**Comment by warpech** [comment]
This is blocked on https://github.com/Starcounter/Guidelines/issues/32

**Comment by tomalec** [comment]
This is needed to point to correct `<script>` when HTML Imports polyfill is used.

For V1 version of webcomponentsjs polyfill, where HTML Imports are not 100% standard compatible, this may not be needed.
There will be no imported document, the entire fragment will be stamped into `<link>` in the main document.

**Comment by warpech** [comment]
As checked in HTML Imports polyfill that is part of WC.js V1, `document._currentScript` is not needed. I will add a PR to remove that.

**Comment by warpech** [comment]
@kegor could you please do that? I can help you set up the environment for developing custom elements if needed.

**Comment by kegor** [comment]
Closed, because it was merged into #42.
